integrations/svelte.md

```diff
@@ -29,6 +29,28 @@ Add `"svelte.plugin.css.diagnostics.enable": false` to your VS Code configuratio
 
 Add <kbd>[svelte-windicss-preprocess]</kbd> to your bundler configuration.
 
+### Svelte Kit (As of 1.0.0-next.100)
```

Comment:
I would prefer if the instruction part is actually moved down under the vanilla Svelte section please